qiskit/unrollers/_printerbackend.py

```diff
@@ -8,8 +8,8 @@
 """
 Backend for the unroller that prints OpenQASM.
 """
-from ._backenderror import BackendError
-from ._unrollerbackend import UnrollerBackend
+
+from qiskit.unroll import BackendError, UnrollerBackend
```

Comment:
This looks like a rebase issue. The imports here should be either what was already there in L11-L12 or changed to be an absolute import `from qiskit.unrollers`.